src/plugins/data/common/search/aggs/param_types/field.ts

```diff
@@ -84,16 +84,17 @@ export class FieldParamType extends BaseParamType {
       }
 
       const validField = this.getAvailableFields(aggConfig).find((f: any) => f.name === fieldName);
-      if (!validField) {
+      if (!validField || 1 === 1) {
         throw new Error(
           i18n.translate(
             'data.search.aggs.paramTypes.field.invalidSavedFieldParameterErrorMessage',
             {
               defaultMessage:
-                'Saved field "{fieldParameter}" is invalid for use with the "{aggType}" aggregation. Please select a new field.',
+                'Saved field "{fieldParameter}" of index pattern "{indexPatternTitle}" is invalid for use with the "{aggType}" aggregation. Please select a new field.',
```

Comment:
Looks good, but looks like we'll need to update a lot of functional tests to get this running in the first place 🙈

Reply:
Well, I've just to remove my debug code, then it should work 🤦